mechanics/RELEASE-VERSIONING-PRINCIPLES.md

```diff
@@ -307,7 +307,7 @@ tuple
    </td>
    <td rowspan="3" >2021-03-16
    </td>
-   <td rowspan="3" >1.19.x
+   <td rowspan="3" >(undetermined)
```

Comment:
Perhaps we should open issues for each of these to update it before each release?

cc @abrennan89